### PR TITLE
team page

### PR DESCRIPTION
**Pull request by rishinhk004**

### CODE REVIEW

File: src/Pages/TeamPage/Team.jsx [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+                <div className={styles.social}>
+                    <img className={styles.logo} src="https://s3-alpha-sig.figma.com/img/f450/f8aa/8efd2714f3a4aaccae56c77a519aa596?Expires=1705881600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=SsqKOC8E9WXu9owo67Cu0M193CvxWusMmK~nEuxe2bH6ETo2lSbaWSkVEdg5icZLiKCP~JVGNUOnU9lQrM5LcvTFvs8-aVLsHoCYxKIL5~Kewtaqfx5SwaWMERKeGeFOO-YBGFCgEsXn-lkRPU5Ravo0CDteoA5mnY5CJfvOcofFKd5QTLCzu~qRdfTHpZFsAoz-KreK~skqcoClwJDieJ41eytTo-YJbsUTGa1HIOQOdCMmd8K7Sf71jkHf9XP1P5NkFVROK-9KWlAQDUssDl23Y5H2uWiXT4VYknyorwqYJoBpvk3-PnXoyEIbvZz6gGrR~O6EKr4jJwkTVEAzhw__" alt="instagram" />
+                    <img className={styles.logo} src="https://s3-alpha-sig.figma.com/img/1c7f/bbb4/7f72f4cd9c19126b202afe2584ed4b29?Expires=1705881600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=i1aXekZZobz2lwcS9bGkvY0Eq1G9ciXyxeLWsijHv1IcbrT0jnyxF7m3WSWdeGFZvjerRlW63kc-0CJCe40vIxyjkAuaoGoMaY7dgbmWvI1ZB7bSvwAwZmjzji2p937PXrHDPadnjKyvSnoNoa2OlU8dRJOQBqK28srYVRl4GwtjHPr4QwmDNe9yayQSmybKvso-Y-SgaDepzLe06UfRA34Iaa--hpS-WSpvNwfy6l~dY1~~fO586pLTr5bVVOD4FQRDszss5Om8~9EQdzGMcRXbFAw3EGJ8xbv1CHAtsFuku8B0WSehTx1z2ssolcH5wWo89qBwqvdfeM3BcMO88g__" alt="facebook" />
```
Comment (bj-jiwrajka) [inline src/Pages/TeamPage/Team.jsx]: replace the URLs with the cloudinary links used in footer

File: src/Pages/TeamPage/Team.jsx [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+        {
+            id: 4,
+            team: "CORE",
+        },
+        {
+            id: 5,
+            team: "VOLUNTEERS",
+        },
+
+    ];
```
Comment (bj-jiwrajka) [inline src/Pages/TeamPage/Team.jsx]: make separate json files for faculty, tech team, organisers, etc and use them here

File: src/Pages/TeamPage/Team.module.scss [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+  height: 3rem;
+  width: 10rem;
+
+  .state_layer {
+    align-items: center;
+    display: inline-flex;
+    flex: 0 0 auto;
+    gap: 0.5rem;
+    justify-content: center;
+    padding: 10px 24px;
```
Comment (bj-jiwrajka) [inline src/Pages/TeamPage/Team.module.scss]: replace ```px``` by  ```rem/em```

File: src/Pages/TeamPage/Team.module.scss [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+    right: 11rem;
+  }
+}
+@media only screen and (min-width: 1001px) {
+  .teamname_body {
+    left: 23%;
+  }
+}
+@media only screen and (max-width: 400px) {
+}
```
Comment (bj-jiwrajka) [inline src/Pages/TeamPage/Team.module.scss]: remove the redundant code and try to choose uniform breakpoints, _take reference from other components_

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

**Comment by byt3h3ad** [comment]
image
github icon looks weird. 


image
alignment issue again.

File: src/Pages/TeamPage/TeamPage.module.scss [hunk 5]
```diff
@@ -0,0 +1,215 @@
+$fontcolor: #ffffff;
+
+.teamsCont {
+  width: 100%;
+  overflow-x: hidden;
```
Comment (byt3h3ad) [inline src/Pages/TeamPage/TeamPage.module.scss:5]: this overflow hidden is hampering the functioning of  react-scroll. so either remove this or don't use react-scroll.

**Pull request merged**